packages/components/src/components/segmented-control/segmented-control/segmented-control.tsx

```diff
@@ -47,6 +49,9 @@ export class SegmentedControl {
   /** Sets the initial value of the segmented-control. */
   @Prop() public value?: string | number;
 
+  /** Sets the width of the items. */
+  @Prop() public columns?: BreakpointCustomizable<number | 'auto'> = 'auto';
```

Comment:
Is number a good choice? What about numbers like `-1`, `0` or large ones like `27`?